sale_project_timesheet_by_seniority/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
Please specify license

Reply:
I am never sure if it is really required or not as there is no code in the file but just imports ?
But added anyway.

Reply:
See template https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
Indeed :+1: